lib/actions/InitAction.js

```diff
@@ -69,6 +67,29 @@ class InitAction {
     if (this.options.appId) return cb(null, this.options.appId)
     prompt([{type: 'input', name: 'appId', message: 'Enter your Sandbox App ID:'}]).then(answers => cb(null, answers.appId))
   }
+
+  permitDeletion (prompt, cb) {
+    prompt({type: 'input', name: 'overwrite', message: 'Do you really want to overwrite your current application? (y/n)'}).then(answers => {
+      return cb(null, answers.overwrite === 'y')
+    })
+  }
+
+  initApplication (appId, userSettings, reset, cb) {
+    new DcHttpClient(userSettings).getApplicationData(appId, (err) => {
+      if (err) {
+        logger.debug(err)
+        if (cb) return cb(new Error(`The application ${appId} is not available or permissions are missing (message: ${err.message}). Please check the application at developer.shopgate.com!`))
+      }
+      if (reset) this.resetApplication()
+      AppSettings.setInstance(new AppSettings().setId(appId).save().init())
+      cb(null, appId)
+    })
+  }
+
+  resetApplication () {
+    const appPath = process.env.APP_PATH ? process.env.APP_PATH : ''
+    fsEx.emptyDirSync(path.join(appPath, '.sgcloud'))
```

Comment:
I guess these are the only files that need to be deleted, as the others get overwritten anyway?